### ISSUE TITLE
Fix latest install of package

### ISSUE THREAD

**Issue by bkuhlmann** [hanami/assets-js#14]
## Why

Installing the *latest* version of `hanami-assets` installs the Beta version instead of the Release Candidate. Also, this is semi-related to Issue #13 where you'll get the following error: *Type error with `assets.run` not a function*.

## How

Run the following to recreate:

``` bash
npm install hanami-assets@latest
$EDITOR node_modules/hanami-assets/package.json
```

Notice that you'll have the following:

``` json
"version": "2.1.0-beta.2"
```

The problem is that the *latest* version is actually: `2.1.0-rc.1`. Using `2.1.0-beta.2` also causes the following error in your console:

``` 
await assets.run();
TypeError: assets.run is not a function
```

...when using:

``` bash
node config/assets.mjs
```

**Comment by timriley** [comment]
Thanks for the report, @bkuhlmann.

_In general_, I'm not sure we'd want to ever tag prereleases (alpha, beta or rc) as "latest"; those feel like they should require more intentional user opt-in. This is an awkward moment, yes, since there's been no proper stable release of hanami-assets yet, but once this is done, we'll want to prioritise for the experience of people running stable releases rather than prereleases.

However, this does make me feel like we should have a clear and consistent bit of doco around publishing this npm package. Why, for instance, was beta2 tagged as "latest" but rc1 wasn't?

<img width="783" alt="Screenshot 2023-11-06 at 10 52 18 am" src="https://github.com/hanami/assets-js/assets/3134/a03c82e1-6dce-4653-8bc6-e77bc0967281">

We'll want to make this clear for future releases.

**Comment by bkuhlmann** [comment]
>  I'm not sure we'd want to ever tag prereleases (alpha, beta or rc) as "latest".

I suppose that makes sense. Yet, technically, a pre-release is the "latest". Knowing you can use "latest" helps with scripting -- as a downstream consumer -- because if you don't want to constantly manually update, then there is "latest" to fallback on which reduces maintenance burden. You can always pin if you don't want the "latest" too. That said, I avoid pre-releases even though the [Semantic Versioning](https://semver.org) allows it because they tend to get messy and confusing.

> Why, for instance, was beta2 tagged as "latest" but rc1 wasn't?

Agreed. Super confusing and surprising behavior. Would definitely be nice to have a consistent strategy moving forward.

**Comment by bkuhlmann** [comment]
:information_source: Closing this in favor of picking up 2.1.0 shortly. Plus I'm able to use RC 1 and RC 2 (while being careful to respect the different RC suffixes).
